app/src/editor/shared/components/modules/Chart.jsx

```diff
@@ -0,0 +1,353 @@
+import React from 'react'
+import cx from 'classnames'
+import throttle from 'lodash/throttle'
+import debounce from 'lodash/debounce'
+import Highcharts from 'highcharts/highstock'
+
+import HighchartsReact from 'highcharts-react-official'
+import ModuleSubscription from '$editor/shared/components/ModuleSubscription'
+import SvgIcon from '$shared/components/SvgIcon'
+
+import styles from './Chart.pcss'
+
+const rangeConfig = {
+    All: 'all',
+    '1 month': 30 * 24 * 60 * 60 * 1000,
```

Comment:
I think you'll need to separate the keys and the labels for translation.

Reply:
Ah yeah you're right.